src/BusyImage.tsx

```diff
@@ -82,14 +82,15 @@ const BusyImage = (props: BusyImageProps) => {
 
     // set the label position relative to the BusyImg
     const maxLabelWidth = labelRef.current !== null ? labelRef.current.offsetWidth : 0;
+    const labelHeight = labelRef.current !== null ? labelRef.current.offsetHeight : 0;
 
     if (ev.clientX + ev.currentTarget.scrollLeft > ev.currentTarget.scrollWidth/2){
       setHoveredLabelX(ev.clientX - rect.x + ev.currentTarget.scrollLeft - maxLabelWidth);
     } else {
       setHoveredLabelX(ev.clientX - rect.x + ev.currentTarget.scrollLeft + 10);
     };
     if (ev.clientY - rect.y > ev.currentTarget.scrollHeight/2){
-      setHoveredLabelY(ev.clientY - + rect.y - 100);
+      setHoveredLabelY(ev.clientY - rect.y - labelHeight);
     } else {
       setHoveredLabelY(ev.clientY - rect.y + 10);
```

Comment:
I think both upper and lower halves of the image should take the label height into account. There's less gap in the upper section, but it's inconsistent depending on if there's extra data.